code/edgeidentity/src/main/java/com/adobe/marketing/mobile/edge/identity/IdentityItem.java

```diff
@@ -42,15 +43,15 @@ public final class IdentityItem {
 	 * @param id                 id for the item; should not be null
```

Comment:
id for the item; should not be null or empty 

Also, don't forget to update the api reference page if we don't already cover this